### ISSUE TITLE
Write admin documentation (how to set up & update Paperwork)

### ISSUE THREAD

**Issue by devilx** [twostairs/paperwork#32]
As the title says.

**Comment by devilx** [comment]
@DeviantEng already wrote a blog entry on installing Paperwork on CentOS 7. Also did I extend the `README.md` be more specific regarding installation and updating. Still, Paperwork lacks of an official Help-page (`/app/help/...`) dedicated to this topic.

**Comment by jamesrascal** [comment]
I created a guide for Ubuntu/Debian that would help and beginner deploy it on a clean server.

[Wiki Link] (https://github.com/twostairs/paperwork/wiki/Installation---Ubuntu-Debian)

I am working on writing a guide for additional OS support.

**Comment by devilx** [comment]
@jamesrascal hey, that's cool! Thank you! I'll publish that on the Paperwork social media channels.

**Comment by devilx** [comment]
@DeviantEng maybe you could help out with that CentOS part, since you've already blogged a great documentation there?

**Comment by DeviantEng** [comment]
@devilx If I find the time this week, I'll port over my blog to a wiki page here.

**Comment by devilx** [comment]
Thank you in advance!

**Comment by DeviantEng** [comment]
@devilx CentOS page has been added to Wiki, and updated the link on the wiki homepage.
[Installation - CentOS 7](https://github.com/twostairs/paperwork/wiki/Installation-----CentOS)

Got to say man, this project has really advanced from where I first used it back in Sept/Oct.  I had only used this in testing, but I think I'm going to give it a run to replace my Tagspaces setup on top of OwnCloud (keep OwnCloud, but ditch Tagspaces), since my current setup isn't working all that great for my workflow.  

Keep up the awesome work!
